src/Gui/FileHandler.h

```diff
@@ -1,50 +1,61 @@
- *                                                                         *
- *   You should have received a copy of the GNU Library General Public     *
- *   License along with this library; see the file COPYING.LIB. If not,    *
- *   write to the Free Software Foundation, Inc., 59 Temple Place,         *
- *   Suite 330, Boston, MA  02111-1307, USA                                *
- *                                                                         *
- ***************************************************************************/
-
-#ifndef IMAGE_WORKBENCH_H
-#define IMAGE_WORKBENCH_H
-
-#include <Gui/Workbench.h>
-#include <Mod/Image/ImageGlobal.h>
-
-namespace ImageGui {
-
-/**
- * @author Werner Mayer
- */
-class ImageGuiExport Workbench : public Gui::StdWorkbench
-{
-    TYPESYSTEM_HEADER_WITH_OVERRIDE();
-
-public:
-  Workbench();
-  ~Workbench() override;
-
-protected:
-  Gui::ToolBarItem* setupToolBars() const override;
-  Gui::ToolBarItem* setupCommandBars() const override;
-};
-
-} // namespace ImageGui
-
-
-#endif // IMAGE_WORKBENCH_H 
+/***************************************************************************
+ *   Copyright (c) 2023 Werner Mayer <wmayer[at]users.sourceforge.net>     *
+ *                                                                         *
+ *   This file is part of the FreeCAD CAx development system.              *
+ *                                                                         *
+ *   This library is free software; you can redistribute it and/or         *
+ *   modify it under the terms of the GNU Library General Public           *
+ *   License as published by the Free Software Foundation; either          *
+ *   version 2 of the License, or (at your option) any later version.      *
+ *                                                                         *
+ *   This library  is distributed in the hope that it will be useful,      *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU Library General Public License for more details.                  *
+ *                                                                         *
+ *   You should have received a copy of the GNU Library General Public     *
+ *   License along with this library; see the file COPYING.LIB. If not,    *
+ *   write to the Free Software Foundation, Inc., 59 Temple Place,         *
+ *   Suite 330, Boston, MA  02111-1307, USA                                *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef GUI_FILE_HANDLER_H
+#define GUI_FILE_HANDLER_H
+
+#include <QString>
+#include <string>
+
+namespace App {
+class Document;
+}
+
+namespace Gui {
+
+class FileHandler
+{
+public:
+    FileHandler(const QString& filename);
```

Comment:
```suggestion
    explicit FileHandler(const QString& filename);
```

Reply:
lint told me that too but I forgot to add it.

Reply:
Fixed